File: app/src/main/java/com/darklycoder/wifitool/MyApplication.java
package com.darklycoder.wifitool;

import android.app.Application;

import com.darklycoder.wifitool.lib.WiFiConfig;
import com.darklycoder.wifitool.lib.WiFiModule;

public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        //初始化
        WiFiConfig config = new WiFiConfig.Builder()
                .setTimeOut(1000 * 20)
                .build();
        WiFiModule.getInstance().setConfig(config).init(this);
    }

}

File: app/src/main/java/com/darklycoder/wifitool/MyApplication.kt
package com.darklycoder.wifitool

import android.app.Application

import com.darklycoder.wifitool.lib.WiFiConfig
import com.darklycoder.wifitool.lib.WiFiModule

class MyApplication : Application() {

    override fun onCreate() {
        super.onCreate()

        //初始化
        val config = WiFiConfig.Builder()
                .setTimeOut((1000 * 20).toLong())
                .build()

        WiFiModule.setConfig(config).init(this)
    }

}
